### PR TITLE
Use Jaeger-Thrift exporter by default

### PR DESCRIPTION
**Pull request by iNikem**

### CODE REVIEW

File: agent/src/main/java/com/splunk/opentelemetry/SplunkAgent.java [hunk 1]
```diff
@@ -28,10 +28,10 @@ public static void premain(final String agentArgs, final Instrumentation inst) {
   }
 
   public static void agentmain(final String agentArgs, final Instrumentation inst) {
-    setDefaultConfig("otel.exporter", "zipkin");
+    setDefaultConfig("otel.exporter", "jaeger-thrift");
     // http://localhost:9080/v1/trace is the default endpoint for SmartAgent
-    // http://localhost:9411/api/v2/spans is the default endpoint for otel-collector
-    setDefaultConfig("otel.exporter.zipkin.endpoint", "http://localhost:9080/v1/trace");
+    // http://localhost:14268/api/traces is the default endpoint for otel-collector
+    setDefaultConfig("otel.exporter.jaeger.endpoint", "http://localhost:9080/v1/trace");
```
Comment (mateuszrzeszutek) [inline agent/src/main/java/com/splunk/opentelemetry/SplunkAgent.java:34]: Is the Smart Agent endpoint exactly the same for zipkin and jaeger?

**Review by mateuszrzeszutek — APPROVED** [review]

File: agent/src/main/java/com/splunk/opentelemetry/SplunkAgent.java (hunk 1 above)
Comment (iNikem) [reply agent/src/main/java/com/splunk/opentelemetry/SplunkAgent.java:34]: At least that what I understood from SA documentation. @flands can you confirm?

Comment (flands) [reply agent/src/main/java/com/splunk/opentelemetry/SplunkAgent.java:34]: Yes same port

**Review by flands — APPROVED** [review]
LGTM -- please ensure the release notes make this very clear! bump the minor version for this release.

**Pull request merged**